examples/ui/font_atlas_debug.rs

```diff
@@ -88,6 +88,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>, mut state: ResM
                 style: TextStyle {
                     font_size: 60.0,
                     color: Color::WHITE,
+                    ..Default::default()
```

Comment:
This doesn't work how it used to. It stops adding new glyphs to the atlas after a certain (random) point in time. Probably because it hits a random character that breaks part of the the glyph generation process?

Reply:
This example was breaking because the `size` field of the text component's style was set to be [250px, 60px].  

The `size` is being used to limit the bounds that `glyph_brush_layout` generates the glyphs in, and its behavior in this case is to ... Not generate glyphs that are out of bounds (probably by early returning btw, I haven't checked the sources). 

I fixed the example by removing the size of the component, I do not think that this is a bug per se.

Reply:
yeah that that seems like a desirable property. carry on!